src/inc/msquic.h

```diff
@@ -482,6 +482,8 @@ typedef struct QUIC_STATISTICS_V2 {
 
     uint32_t SendCongestionWindow;          // Congestion window size
 
+    uint32_t ChangeSrcCidCount;             // Number of times the source CID changed.
```

Comment:
After thinking about it, I like `SrcCidUpdate*` instead of `ChangeSrcCid*`. I'd appreciate it if you would update. Thanks!

Reply:
Updated!